cert/cert-renewal/src/policy.rs

```diff
@@ -66,12 +66,29 @@ impl<'de> serde::Deserialize<'de> for Policy {
     {
         let mut policy: String = serde::Deserialize::deserialize(deserializer)?;
 
-        let unit = if let Some(unit) = policy.pop() {
-            unit
+        let last = if let Some(last) = policy.pop() {
+            last
         } else {
             return Err(serde::de::Error::custom("policy not specified"));
         };
 
+        let mut unit = last.to_string();
+
+        // Parse for "min" or "day".
+        if last == 'n' || last == 'y' {
+            if let Some(last) = policy.pop() {
+                unit = last.to_string() + &unit;
```

Comment:
`unit.insert(0, last);`

But also, something like `policy.len().checked_sub(3).map(|prefix_len| policy.split_at(prefix_len))` will give you the prefix and suffix in one shot.

Reply:
Ah, right.